.eslintignore

```diff
@@ -0,0 +1 @@
+.ncurc.js
```

Comment:
I usually just `ln -s .gitignore .eslintignore` and then commit the symbolic link.

Reply:
The link is an interesting approach, but I don't know if you the untracked files are always the ones you don't want linted. I updated this to just `dist/`, since that's the only folder that has JS files in that list

Reply:
Normally at least `coverage` should be included, but as we're not using the HTML or iconv reporter (and hopefully keeping 100% coverage), we shouldn't need to dig into such reports with the text ones being sufficient.

Reply:
I also meant to have `!.*.js` added to ensure linting the hidden files since they are ignored in ESLint by default.

We should at least add that one and also `.ignore` so people have a place to put JavaScript tests and junk without getting them linted.

Reply:
OK, haven't really come across the `.ignore` before, but I've added it in with the other one. Add some comments so it's more explicit for the next person. I haven't come across the `coverage` folder in this repo, just the `.nyc_output`, but that is all JSON files

Reply:
Re: `.ignore`, `.idea` would work fine too--just to have a place for experiments or such. Re: `coverage` you'll only see it if you run against an html or iconv reporter, but as we don't have scripts for that and don't need it, that's fine.